e2e/README.md

```diff
@@ -0,0 +1,17 @@
+# How to debug E2E tests on CI
+
+1. Go to E2E repository https://github.com/opencrvs/e2e
+2. Find the `Deploy & run E2E` action run that failed
+3. Locate the failing test(s)
+4. Under `upload-artifact` step, download the artifact
+5. Unzip the downloaded artifact
+6. Open the `index.html` file in browser
```

Comment:
There is this command which @jamil314 had shown us: `npx playwright show-report {location/to/unzipped/artifact}`
Running this should take us directly to step 13